docs/reference/alpha-vector-database.md

```diff
@@ -11,16 +11,29 @@ Below are supported vector databases and implemented features:
 |-----------------|-----------|----------|
 | Pgvector        | [x]       | [ ]      |
 | Elasticsearch   | [x]       | [x]      |
-| Milvus          | [ ]       | [ ]      |
+| Milvus          | [x]       | [x]      |
 | Faiss           | [ ]       | [ ]      |
 | SQLite          | [x]       | [ ]      |
 | Qdrant          | [x]       | [x]      |
 
 Note: SQLite is in limited access and only working on Python 3.10. It will be updated as [sqlite_vec](https://github.com/asg017/sqlite-vec/) progresses.
 
-## Example
+{% hint style="danger" %}
+We will be deprecating the `retrieve_online_documents` method in the SDK in the future. 
+We recommend using the `retrieve_online_documents_v2` method instead, which offers easier vector index configuration 
+directly in the Feature View and the ability to retrieve standard features alongside your vector embeddings for richer context injection. 
 
-See [https://github.com/feast-dev/feast-workshop/blob/rag/module_4_rag](https://github.com/feast-dev/feast-workshop/blob/rag/module_4_rag) for an example on how to use vector database.
+Long term we will collapse the two methods into one, but for now, we recommend using the `retrieve_online_documents_v2` method.
+Beyond that, we will then have `retrieve_online_documents` and `retrieve_online_documents_v2` simply point to `get_online_features` for 
+backwards compatibility and the adopt industry standard naming conventions.
+{% endhint %}
+
+**Note**: Milvus implements the v2 `retrieve_online_documents_v2` method in the SDK. This will be the longer-term solution so that Data Scientists can easily enable vector similarity search by just flipping a flag.
```

Comment:
should we add another column "V2 support" in the above table, to clarify this point?

Reply:
Done!